Chapter2_ProgramStructure/Chapter 2 #3.js

```diff
@@ -0,0 +1,31 @@
+function f2(size)
+{
+    var i;
+    var j;
+    var t="";
+    for(i=0;i<size;i++)
+    {
+        for(j=0;j<size;j++)
+        {
```

Comment:
How would you re-write the body of this inner for loop if I asked you to test just one condition? -- 
if ((x + y) % 2 == 0)